### PR TITLE
Changed the Application namespace

### PR DESCRIPTION
**Pull request by andreluizmachado**
Renamed `App` to `OAT\SimpleRoster`

Ticket: RO-49

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - Raised minimum required PHP version from `7.2` to `7.3`.
 - Upgraded Symfony framework version from `4` to `5`.
 - Changed `user-ids` and `line-item-ids` input options of [roster:doctrine-result-cache:warmup](docs/cli/doctrine-result-cache-warmer-command.md) command to `usernames` and `line-item-slugs`.
+- Application namespace now is `OAT\SimpleRoster` instead of `App`
```
Comment (Skyld) [inline CHANGELOG.md:12]: Could you please rephrase it as a sentence, like the other changes?

File: tests/Functional/Command/Ingester/IngesterCommandTest.php [hunk 2]
```diff
@@ -108,7 +108,9 @@ public function testLocalIngestionFailure(): void
         );
 
         self::assertStringContainsString(
-            'Argument 1 passed to App\Entity\Infrastructure::setLtiSecret() must be of the type string, null given',
+            <<<MESSAGE_ERROR
```
Comment (Skyld) [inline tests/Functional/Command/Ingester/IngesterCommandTest.php:111]: What is this? Could you please break in into multiple lines with concatenation?

File: tests/Integration/Ingester/Ingester/InfrastructureIngesterTest.php [hunk 3]
```diff
@@ -114,7 +114,9 @@ public function testIngestWithInvalidSource(): void
         );
 
         self::assertStringContainsString(
-            'Argument 1 passed to App\Entity\Infrastructure::setLtiSecret() must be of the type string, null given,',
+            <<<MESSAGE_ERROR
```
Comment (Skyld) [inline tests/Integration/Ingester/Ingester/InfrastructureIngesterTest.php:117]: Same here please

**Review by Skyld — CHANGES_REQUESTED** [review]
Looks nice 👍  I left a couple of small comments. Also, could you please update the title of the PR describing what was done? Thanks!

**Review by luisfn — COMMENTED** [review]
Nothing to add from what @Skyld said.
Fix the small issues and LGTM

**Comment by uncleempty** [comment]
Same frome me. LGTM after fixing Dani comments

**Review by luisfn — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +20,7 @@
 ### Fixed
 - Fixed bug in `DoctrineResultCacheWarmerCommand` where lack of order by clause caused wrong pagination with PostgreSQL.
 - Fixed `OAuthSignatureValidationSubscriber` to read LTI credentials from configuration instead of database.
+- Fixed Code Style issues in `InfrastructureIngesterTest` and `IngesterCommandTest` files
```
Comment (Skyld) [inline CHANGELOG.md:23]: Why did you add this under 1.8.1? I suppose it was by mistake

File: CHANGELOG.md (hunk 1 above)
Comment (Skyld) [inline CHANGELOG.md:12]: Why did you resolve this? I don't see any changes

File: tests/Functional/Command/Ingester/IngesterCommandTest.php [hunk 5]
```diff
@@ -107,8 +107,11 @@ public function testLocalIngestionFailure(): void
             $this->normalizeDisplay($this->commandTester->getDisplay())
         );
 
+        $messageError = 'Argument 1 passed to OAT\SimpleRoster\Entity\Infrastructure::setLtiSecret() '
```
Comment (Skyld) [inline tests/Functional/Command/Ingester/IngesterCommandTest.php:110]: Why not `$errorMessage`? Please rename variable

File: tests/Integration/Ingester/Ingester/InfrastructureIngesterTest.php [hunk 6]
```diff
@@ -113,8 +113,11 @@ public function testIngestWithInvalidSource(): void
             $failure->getData()
         );
 
+        $messageError = 'Argument 1 passed to OAT\SimpleRoster\Entity\Infrastructure::setLtiSecret() '
```
Comment (Skyld) [inline tests/Integration/Ingester/Ingester/InfrastructureIngesterTest.php:116]: Same here

**Review by Skyld — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (andreluizmachado) [reply CHANGELOG.md]: Sorry. My bad. Fixed :)

File: CHANGELOG.md (hunk 1 above)
Comment (andreluizmachado) [reply CHANGELOG.md]: About this comment. I do not know if this is enough. Please let me know in case it's not ok.

**Review by Skyld — APPROVED** [review]
- [x] New code is covered by tests
- [x] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues
- [x] Feature is working correctly on my local machine
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Changelog is updated according to changes
- [x] Documentation is updated according to changes
